src/codeview.js

```diff
@@ -586,6 +619,12 @@ var Codeview = GObject.registerClass({
 
         if (this._numErrors !== results.length) {
             this._numErrors = results.length;
+
+            if (this._dbus) {
+                const variant = new GLib.Variant('b', !!this._numErrors);
+                this._dbus.emit_property_changed('errors', variant);
```

Comment:
Notify the GObject property change. too.

Reply:
`this.notify('errors');`'

Reply:
done